examples_tests/61.BlitFilterTest/blit/formats/rgb10_a2.glsl

```diff
@@ -0,0 +1,19 @@
+#ifndef _NBL_GLSL_BLIT_RGB10_A2_INCLUDED_
+#define _NBL_GLSL_BLIT_RGB10_A2_INCLUDED_
+
+#include <nbl/builtin/glsl/limits/numeric.glsl> // Todo(achal): Can remove this after merging with `master`
+#include <nbl/builtin/glsl/format/encode.glsl>
+#include <nbl/builtin/glsl/format/decode.glsl>
+
+uvec4 nbl_glsl_blit_formats_encode(in vec4 value)
+{
+	const uint encoded = nbl_glsl_encodeRGB10A2_UNORM(value);
+	return uvec4(encoded, 0, 0, 0);
+}
+
+vec4 nbl_glsl_blit_formats_decode(in uvec4 value)
+{
+	return nbl_glsl_decodeRGB10A2_UNORM(value.x);
+}
```

Comment:
whatabout snorm variant?

Reply:
There is none.
Reference: https://www.khronos.org/opengl/wiki/Image_Load_Store#Images_in_the_context

Reply:
but these are your "software encoded" stores, so it shouldn't matter if it exists for image storage or not

Reply:
> There is none. Reference: https://www.khronos.org/opengl/wiki/Image_Load_Store#Images_in_the_context

ah yes, me dumb....